AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -245,6 +246,15 @@ extension AWSInitialSyncOrchestrator {
            case .unauthorized = AppSyncErrorType(errorTypeValue) {
             return true
         }
+        
+        // Check is API error is of unauthorized type
+        if case let .api(amplifyError, _) = datastoreError,
+           let apiError = amplifyError as? APIError {
+            if apiError.isUnauthorized() {
+                return true
+            }
```

Comment:
Here, instead of calling isUnauthorized, can we just check `.httpStatus` and `.operationError`? for httpStatus -> 401/403 return true, for operationError -> errorDescription contains "unauthorized" return true